tutor/src/screens/assignment-review/ux.js

```diff
@@ -324,6 +324,10 @@ export default class AssignmentReviewUX {
     return Boolean(['reading', 'homework'].includes(this.planScores.type));
   }
 
+  @computed get isExternal() {
+    return Boolean(this.planScores.type === 'external');
+  }
+
```

Comment:
I typically prefer to move stuff like this into the model, since it doesn't really have any relation the the UI state.  I think I already added it in #3045